tests/src/Unit/PermissionEventTest.php

```diff
@@ -445,6 +445,8 @@ public function testOffsetSet(array $permissions, $entity_type_id, $bundle_id, a
   public function testOffsetSetInvalidPermission($key, $permission) {
     $event = new PermissionEvent($this->randomMachineName(), $this->randomMachineName(), []);
     $event[$key] = $permission;
+
+    return $event;
```

Comment:
This should not be here, this test should not return any values. PHP_CodeSniffer cannot understand here that the variable is actually used in that it triggers an exception.

Let's add the following comments around it to disable this check:

```
    // Avoid false positive in coding standards checking. The variable is actually used for the test.
    // phpcs:disable DrupalPractice.CodeAnalysis.VariableAnalysis.UnusedVariable
    $event = new PermissionEvent($this->randomMachineName(), $this->randomMachineName(), []);
    $event[$key] = $permission;
    // phpcs:enable DrupalPractice.CodeAnalysis.VariableAnalysis.UnusedVariable
```